client/src/components/ProfileSideBar/ProfileSideBar.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { Box, Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+
+const ProfileSideBar = (): JSX.Element => {
+  const [active, setActive] = useState('Edit Profile');
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.profileSidebar}>
+      {urls.map((url) => (
+        <Link key={url.link} to={`/profile-settings${url.link}`} className={classes.link}>
+          <Button
```

Comment:
The MUI Button and react-router's Link can be combined as below:

Eg: `<Button component={Link} to='/profile-settings${url.link}' .....>`